store/sql_status_store.go

```diff
@@ -89,14 +90,46 @@ func (s SqlStatusStore) Get(userId string) StoreChannel {
 	return storeChannel
 }
 
+func (s SqlStatusStore) GetByIds(userIds []string) StoreChannel {
```

Comment:
I don't think we need caching here since before we hit this it checks if the statuses are already cached in the status cache